.codebuild/scripts/e2e_test_windows.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# set exit on error to true
+set -e
+
+source ./shared-scripts.sh && _setupE2ETestsWindows
+codebuild-breakpoint
```

Comment:
This is keeping the same behavior as E2E tests on linux. I'm not sure `codebuild-breakpoint` is strictly needed on either.

Reply:
This is used when you want to run the tests in debug mode. I think the script is called `yarn-e2e-debug`.